src/lib/decodeTx.test.ts

```diff
@@ -14,6 +14,7 @@ describe(decodeTx.name, () => {
     );
 
     expect(actual.tx).toBeA(Transaction);
+    expect(actual.tx).toMatchSnapshot();
     expect(actual.senderAddr).toEqual(
       '0x67835910d32600471f388a137bbff3eb07993c04',
     );
```

Comment:
```suggestion
    expect(actual.tx).toBeA(Transaction);
    expect(actual.senderAddr).toEqual(
      '0x67835910d32600471f388a137bbff3eb07993c04',
    );
    expect(actual.tx).toMatchSnapshot();
```

Let's have a convention to have snapshot assertions as the last assertion of the group.
They usually produce less readable output and serve as a last resort/catch all assertion.